### DISCUSSION TITLE
[Support]: How to set fps ?

### DISCUSSION THREAD

**Discussion by atv2016** [blakeblackshear/frigate#10095] [General Support]
### Describe the problem you are having

I am looking to optimise one 1080P camera to recognise license plates and I was wondering if the only place to configure fps is under detect? As in:

```
    detect:
      width: 1920
      height: 1080
      fps: 5
```

If so, what is the maximum I can configure here? If my NVR is set to 25, can I set this to the same?
I ask because if I look in system I see:

Process | P-ID | FPS | CPU % | Memory %
-- | -- | -- | -- | --
ffmpeg | 636 | 30.2 | 15.3% | 0.6%
Capture | 630 | 14.5 | 17.5% | 0.6%
Detect | 500 | 0.9 (22.7 skipped) | 5.2% | 0.7%

Which shows 22.7 skips fps for capture, and detect is still around 0.9. So I was wondering if I need to set the FPS for record as well. It does look like ffmpeg is honouring the new fps.

Finally, is debug capped to 5 fps and would improve_contrast help with recognising license plates? I'm thinking of forcing this one specific camera to night vision to get better focus.

Thanks,

### Version

Latest beta

### Frigate config file

```yaml
Only above config is relevant.
```

### Relevant log output

```shell
na
```

### FFprobe output from your camera

```shell
FFPROBE OUTPUT
Stream 0:
Return Code: 0

Video:

Codec: H.264 / AVC / MPEG-4 AVC / MPEG-4 part 10
Resolution: 1920x1080
FPS: 12/1
```

### Operating system

Other Linux

### Install method

Docker Compose

### Coral version

USB

### Network connection

Wired

### Camera make and model

SANSCO

**Comment by NickM-27** [comment]
there is no reason to run detect fps higher than 10 and in most cases 5 is sufficient. This fps has nothing to do with recordings which is copied directly from the stream
